getting-started.md

```diff
@@ -57,14 +57,14 @@ from [source](https://github.com/appium/appium-doctor)), then run
 `appium-doctor` and supply the `--ios` or `--android` flags to verify that all
 of the dependencies are set up correctly.
 
-You also need to download the Appium client for your language so you can write tests. The Appium clients are simple extensions to the WebDriver clients. You can see the list of clients and links to download instructions at the  [Appium clients list](/slate/en/master/#appium-clients.md).
+You also need to download the Appium client for your language so you can write tests. The Appium clients are simple extensions to the WebDriver clients. You can see the list of clients and links to download instructions at the  [Appium clients list](/docs/en/about-appium/appium-clients/).
 
 #### iOS Requirements
 
 * Mac OS X 10.12 recommended
 * XCode 8 recommended
 * Apple Developer Tools (iPhone simulator SDK, command line tools)
-*  [Ensure you read our documentation on setting yourself up for iOS testing!](/slate/en/master/#running-on-osx.md)
+*  [Ensure you read our documentation on setting yourself up for iOS testing!](http://127.0.0.1:4000/docs/en/drivers/ios-xcuitest/)
```

Comment:
Do you mean `/docs/en/drivers/ios-xcuitest.md`? (Don't need `http://127.0.0.1:4000`?)

Reply:
Yes. That one slipped through. Thanks!